.github/workflows/release.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
   build_sdist:
     name: Build SDist
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Version numbers show up in a lot of places. How do we ensure that we update all of them when we update? Can dependabot take care of it?

Reply:
I don't know. But we do *not* want to move to the latest version anyway?

Reply:
But we want to stick with an older image version, not update?